pkgs/by-name/gi/git-smash/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  installShellFiles,
+  makeWrapper,
+  fuzzyMatcherPackage ? fzf, # can use either fzf or skim as a fuzzy matcher
```

Comment:
```suggestion
```
we should just rely on PATH and not trigger useless rebuilds like this

Reply:
Ok then, changed as proposed. I don't really get the "trigger useless rebuilds" part. Could you please explain this?